docs/10_functions_description.md

```diff
@@ -116,19 +146,31 @@ If you want to know more on how to use it check its help section and the `CPP_ge
 
 In brief, there are several actions you can execute with this function.
 
--   init: initialize the buffer for key presses on a given device (you can also specify the keys of interest that should be listened to).
--   start: start listening to the key presses (carefully insert into your script - where do you want to start buffering the responses).
+-   init: initialize the buffer for key presses on a given device (you can also 
+specify the keys of interest that should be listened to).
+-   start: start listening to the key presses (carefully insert into your script - 
+where do you want to start buffering the responses).
 -   check: till that point, it will check the buffer for all key presses.
     -   It only reports presses on the keys of interest mentioned at initialization.
-    -   It **can** also check for presses on the escape key and abort if the escape key is part of the keys of interest.
--   flush: Empties the buffer of key presses in case you want to discard any previous key presses.
+    -   It **can** also check for presses on the escape key and abort if the escape 
+key is part of the keys of interest.
+-   flush: Empties the buffer of key presses in case you want to discard any 
```

Comment:
```suggestion
-   flush: empties the buffer of key presses in case you want to discard any 
```

Reply:
here 'release' option definition is missing.